14.0 l10n il 1.1

hedshefer opened the pull request:
Description of the issue/feature this PR addresses:

added vat only and vat self invoice that will fit the israely market needs. In addition, removed unused whd taxes and added employees and customers whd account tags

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/75414).

hedshefer commented:
@jco-odoo 
Here is the PR :
added vat only and vat self invoice that will fit the israely market needs. In addition, removed unused whd taxes and added employees and customers whd account tags
